appcelerator-archive/amp
# Remove functions

## How to test
```
$ ampmake build
$ amp fn
Error: not an amp command: fn
See 'amp --help
$ amp --help

Usage:	amp [OPTIONS] COMMAND [ARGS...]

Deploy, manage, and monitor container stacks and functions.

Examples:
amp version

Options:
  -h, --help            Print usage
  -s, --server string   Specify server (host:port)
  -v, --version         Print version information and quit

Management Commands:
  cluster     Cluster management operations
  org         Organization management operations
  password    Password management operations
  service     Service management operations
  stack       Stack management operations
  team        Team management operations
  user        User management operations

Commands:
  login       Login to account
  logout      Logout of account
  logs        Fetch log entries matching provided criteria
  stats       Display amp statistics
  version     Show amp version information
  whoami      Display currently logged-in user

Run 'amp COMMAND --help' for more information on a command.

```
